Simplenote/PasskeyRegistrationResponse.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import AuthenticationServices
+
+struct PasskeyRegistrationResponse: Encodable {
+    struct Response: Encodable {
+        let clientDataJSON: String
+        let attestationObject: String
+    }
+
+    private let email: String
+    private let id: String
+    private let rawId: String
+    private let type: String
+    private let response: PasskeyRegistrationResponse.Response
+
+    init?(from credentialRegistration: ASAuthorizationPlatformPublicKeyCredentialRegistration) {
+        guard let email = SPAppDelegate.shared().simperium.user?.email,
```

Comment:
We should get the email via parameter (rather than coupling the initializer with the AppDelegate)